tests/extras/pyproject.toml

```diff
@@ -8,6 +8,9 @@ authors = ["Your Name <you@example.com>"]
 python = "^3.6"
 setuptools-scm = { version = "^3.4", extras = ["toml"] }
 
+# Extra with dash in the name should work
+boto3-stubs = {extras = ["cognito-idp"], version = "^1.17.82"}
```

Comment:
If you have better idea for package that could be used for testing here, I'm open for suggestions!